src/core/operator_list.js

```diff
@@ -589,6 +618,14 @@ class OperatorList {
     this._resolved = streamSink ? null : Promise.resolve();
   }
 
+  get isOffscreenCanvasSupported() {
+    return this.optimizer.isOffscreenCanvasSupported;
+  }
+
+  set isOffscreenCanvasSupported(value) {
```

Comment:
Is the getter actually used for anything? Otherwise you want the following instead.
```suggestion
  // eslint-disable-next-line accessor-pairs
  set isOffscreenCanvasSupported(value) {
```